doc/classes/Object.xml

```diff
@@ -487,7 +487,7 @@
 				}
 				[/csharp]
 				[/codeblocks]
-				[b][code]Object.connect()[/code] or [code]Signal.connect()[/code]?[/b]
+				[b][code skip-lint]Object.connect()[/code] or [code skip-lint]Signal.connect()[/code]?[/b]
```

Comment:
Could possibly be:
```suggestion
				[b][method Object.connect] or [method Signal.connect]?[/b]
```
But I guess the former will only show as [connect]() since we're in Object?

Reply:
Perhaps, but I just didn't want to break the original formatting since here it seems to be done on purpose for the title.